src/playstate/updatestats.asm

```diff
@@ -189,7 +201,8 @@ addPointsRaw:
         lda #4
         sta completedLines
 @notTapQuantity:
-        lda holdDownPoints
+        jsr testCrash
```

Comment:
we should probably only call this function when crashMode isn't 0

Reply:
crashMode = 3 is disabled, not 0